zio-http/src/main/scala/zio/http/netty/NettyRuntime.scala

```diff
@@ -14,7 +14,7 @@ private[zio] trait NettyRuntime { self =>
 
   def runtime(ctx: ChannelHandlerContext): Runtime[Any]
 
-  def run(ctx: ChannelHandlerContext, interruptOnClose: Boolean = true)(
+  def run(ctx: ChannelHandlerContext, ensured: () => Unit = () => (), interruptOnClose: Boolean = true)(
```

Comment:
This default parameter will allocate a new lambda at the call site whenever it is not specified. I would instead move the `ensured` to the call site if possible.